### PR TITLE
Gridstack 0.2.5

### PR DESCRIPTION
**Pull request by DavidKDeutsch**
This is a new definition file for the latest version of [Gridstack](https://github.com/troolee/gridstack.js). The existing definition file has been renamed to gridstack-0.2.3.d.ts, per the contribution guidelines.

### CODE REVIEW

File: gridstack/gridstack.d.ts [hunk 1]
```diff
@@ -1,241 +1,347 @@
-﻿// Type definitions for Gridstack
+﻿// Type definitions for Gridstack v0.2.5
 // Project: http://troolee.github.io/gridstack.js/
-// Definitions by: Pascal Senn <https://github.com/PascalSenn/>
```
Comment (vvakame) [inline gridstack/gridstack.d.ts]: please keep original author name. add your name below.
like this https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/jquery/jquery.d.ts#L3

**Comment by vvakame** [comment]
If we can't get review by author. I'll merge this pull request 2 days later. please ping me again.

**Comment by mhegazy** [comment]
@DavidKDeutsch can you keep the name of the other authors please and refresh this PR.

**Comment by mhegazy** [comment]
Thanks for your contribution. This PR has merge conflicts and can not be merged in at the time being. For housekeeping purposes we are closing stale PRs. If you'd still like to merge this code in, please open a new PR that has been merged and rebased with the `master` branch.

**Pull request closed without merging**